### PR TITLE
Manual username + password upload for Quay

### PR DESCRIPTION
**Pull request by mshaposhnik**
Signed-off-by: Max Shaposhnik <mshaposh@redhat.com>

### What does this PR do?
Allows manual username + password  (typically robot account) upload for Quay

### What issues does this PR fix or reference?

https://issues.redhat.com/projects/SVPI/issues/SVPI-107

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ COPY controllers/ controllers/
 # build service
 # Note that we're not running the tests here. Our integration tests depend on a running cluster which would not be
 # available in the docker build.
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o spi-oauth main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GOPRIVATE=* go build -a -o spi-oauth main.go
```
Comment (skabashnyuk) [inline Dockerfile:21]: that can be removed before merging right?

File: samples/access_token_data.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
-  "access_token": "123",
-  "expiry": "123135429034087083",
+  "username": "mshaposhnik+tekton_demo",
+  "access_token": "TPQ8OQSY8CC7SNLKH7T7RFN9G8B8N2ZDNC09M......",
```
Comment (skabashnyuk) [inline samples/access_token_data.json:3]: can we make a separate file with `username` and `access_token` for testing the token endpoint? 
Just do not confuse with optional fields.

Comment (mshaposhnik) [reply samples/access_token_data.json:3]: \+

**Comment by skabashnyuk** [comment]
what about creating separate DTO for token?

File: samples/username_password_data.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "username": "quay+user_demo",
+  "access_token": "TPQ8OQSY8CC7SNLKH7",
+  "expiry": 90340870832134324,
+  "refresh_token": "",
+  "token_type": "bearer"
+}
```
Comment (skabashnyuk) [inline samples/username_password_data.json:7]: what about 
```json
{
  "username": "quay+user_demo",
  "access_token": "TPQ8OQSY8CC7SNLKH7",
}
```

**Review by skabashnyuk — APPROVED** [review]

**Review by metlos — APPROVED** [review]

**Pull request merged**